kata-webhook/Dockerfile

```diff
@@ -5,7 +5,7 @@ FROM golang:latest AS builder
 
 WORKDIR /go/src/kata-pod-annotate
 
-COPY ../vendor ./
+# COPY ../vendor ./
```

Comment:
Note that this reverts of the previous commit
I'd squash the 2 fixes and the "fix indentation" into single "webhook: minor fixes" commit of similar.
Also IIRC subject must include "area:" (e.g. **webhook:** minor fixes) and some text in the commit body to pass the checks (the text you added is in the PR not in the commit)
Thanks @dmaizel !